### PR TITLE
wxpython: Add comma to install array

### PR DESCRIPTION
**Pull request by dj-m**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

Modified install array to include comma after the last item

### CODE REVIEW

File: Formula/wxpython.rb [hunk 1]
```diff
@@ -29,9 +29,9 @@ class Wxpython < Formula
     depends_on :python
     depends_on FrameworkPythonRequirement
   end
-  depends_on "wxmac"
 
   option :universal
+  depends_on "wxmac"
```
Comment (DomT4) [inline Formula/wxpython.rb]: Need an empty line between the last option & first `depends_on`. You can move the other block with `depends_on` within as well:

```ruby
option :universal

if MacOS.version <= :snow_leopard
  depends_on :python
  depends_on FrameworkPythonRequirement
end
depends_on "wxmac"
```

File: Formula/wxpython.rb [hunk 2]
```diff
@@ -60,11 +60,15 @@ def install
...
+      "BUILD_STC=1",
     ]
 
     cd "wxPython" do
       system "python", "setup.py", "install", "--prefix=#{prefix}", *args
     end
   end
+
+  test do
+    system "python -c 'import wx ; print wx.version()'"
```
Comment (DomT4) [inline Formula/wxpython.rb]: Let's assert on the version here as well.
```ruby
output = shell_output("python -c 'import wx ; print wx.version()'")
assert_match version.to_s, output
```

Comment (dj-m) [inline Formula/wxpython.rb]: Is it acceptable to have it as one line? e.g. 

`assert_match version.to_s, shell_output("python -c 'import wx ; print wx.version()'")`

Comment (DomT4) [inline Formula/wxpython.rb]: We prefer to wrap to 80 characters as much as possible. Haven't checked but I'd be surprised if that wasn't over. There are various ways to wrap to the 80, but the block I proposed is the most common one in use, at least till we change our minds 😄.

Comment (dj-m) [inline Formula/wxpython.rb]: Great point! Update's on its way... thanks so much! 👏🏾

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/b9082a9b0daad7e8bda2fde60ab943c3d3186ece. Thank you @dj-m! 😺

**Comment by MikeMcQuaid** [comment]
Nice work @dj-m 😍

**Pull request merged**